### DISCUSSION TITLE
Use i18n with markdown files

### DISCUSSION THREAD

**Discussion by pochat** [nuxt/content#2489] [Q&A]
Hi all,

I hope you are well.

I successfully translated my application using json files, but I can't find how to translate markdown files.

For example, my json files are:

```
locales: [
      {
        code: 'en',
        name: 'English (US)',
        file: '../i18n/en-us.json',
      },
      {
        code: 'fr',
        name: 'French (FR)',
        file: '../i18n/fr-fr.json',
      },

```

I use $('programs.message'), which works well.

`$t('programs.message') // returns "Bonjour" or "Hello"
`

However, the $t doesn't work for markdown files.

For example, my markdown files are:

/content/en
-file1.md
-file2.md

/content/fr
-file1.md
-file2.md

Front matter as:
/en/file1.md
---
locale: 'en'
hello: 'hello'
---

/fr/file1.md
---
locale: 'fr'
hello: 'bonjour'
---

Index.vue as;
```
<script setup lang="ts">

const { t } = useI18n()
const localePath = useLocalePath()

</script>

<template>
  <ContentList path="/en/locales" v-slot="{ list }">
    <div v-for="article in list" :key="article._path">
      <h2>{{ t('article.name') }}</h2>
    </div>
  </ContentList>
</template>

<style scoped>

</style>
```

The problem is the $t or the t don't translate when using the language switcher.

Do you have any ideas on how to make this work?

Thank you for your time.

**Comment by nobkd** [comment]
Maybe you can look into the related issue #1908

**Comment by pochat** [comment]
Hi @nobkd,

Thanks for responding.

I saw that issue yesterday. However, they refer to another topic.

Have a nice day.

**Comment by misbahansori** [comment]
You also need to specify locales on the nuxt content config
```
content: {
    locales: ["en", "fr"],
  },
```

**Comment by pochat** [comment]
Ah, thank you @misbahansori !
